charts/pega/README.md

```diff
@@ -838,4 +838,54 @@ certificates:
     badssl.cer: |
       "-----BEGIN CERTIFICATE-----\n<<certificate content>>\n-----END CERTIFICATE-----\n"
 
-```
\ No newline at end of file
+```
+## Deploying Hazelcast in Client Server Mode (Draft)
```

Comment:
draft?

Reply:
Will remove these mentions as part of a last commit. This was just for declaring that changes are still being made.